examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -0,0 +1,740 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class CupertinoNavigationDemo extends StatelessWidget {
+  static const String routeName = '/cupertino/navigation';
+
+  @override
+  Widget build(BuildContext context) {
+    return new WillPopScope(
+      // Prevent swipe popping of this page. Use explicit exit buttons only.
+      onWillPop: () => new Future<bool>.value(true),
+      child: new CupertinoTabScaffold(
+        tabBar: new CupertinoTabBar(
+          items: const <BottomNavigationBarItem>[
+            const BottomNavigationBarItem(
+              icon: const Icon(CupertinoIcons.home),
+              title: const Text('Tab 1'),
+            ),
+            const BottomNavigationBarItem(
+              icon: const Icon(CupertinoIcons.conversation_bubble),
+              title: const Text('Tab 2'),
+            ),
+            const BottomNavigationBarItem(
+              icon: const Icon(CupertinoIcons.profile_circled),
+              title: const Text('Tab 3'),
```

Comment:
better names for the tabs? like Home, Chat, Profile?

Reply:
Sure